tests/unit/CMakeLists.txt

```diff
@@ -18,35 +18,6 @@ include_directories(BEFORE ${PROJECT_BINARY_DIR})
 include_directories(BEFORE ${CMAKE_CURRENT_SOURCE_DIR})
 
 set(UNIT_TESTS_SRCS
```

Comment:
Why are all of these deleted? Please revert these changes back.

Reply:
Sorry that was sloppy of me... just deleted them to speed up testing.